src/com/lftechnology/java/training/alina/jdbc/demo/EmployeeManagement.java

```diff
@@ -0,0 +1,17 @@
+package com.lftechnology.java.training.alina.jdbc.demo;
+
+import java.sql.SQLException;
+import com.lftechnology.java.training.alina.jdbc.controller.LoginController;
+
+/**
+ * EmployeeManagement consists of employee management functionalities
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class EmployeeManagement {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)